tensorflow/python/ops/nn_ops.py

```diff
@@ -1294,7 +1294,7 @@ def conv3d_transpose(value,
     padding: A string, either `'VALID'` or `'SAME'`. The padding algorithm.
       See the @{tf.nn.convolution$comment here}
     data_format: A string, either `'NDHWC'` or `'NCDHW`' specifying the layout
-      of the input and output tensors. Defaults to `'NDHWC'`.
```

Comment:
If we are making the default globally configurable then it would be ambiguous to specify strings in method signatures, might mislead users visiting the API. In that scenario, we can have NoneType.

All cases considered, in my humble opinion, to match current method signatures of other ops in this script, we should go with NoneType at least for this PR. Hence the phrase "Defaults to NDHWC" wasn't valid in the docstring because that's not the default value even if the op implicitly behaves in the same manner when that value would have been NDHWC.

Rest I would leave the decision on veterans and make modifications accordingly.

CC: @martinwicke